### DISCUSSION TITLE
Remove Silence in Podcasts

### DISCUSSION THREAD

**Discussion by sheika-sage** [LeoKlaus/plappa#171] [Feature Requests]
In PocketCasts I have the Smart Trim Silence Feature always on. How complicated would it be to implement it into the app?

**Comment by LeoKlaus** [comment]
I think I found a way to implement this, but I'll have to check how well it works. I'll let you know once I have more info!

**Reply by timespacedecay** [reply]
Any update on this?  The ability to "trim silence" like Overcast, PocketCasts, etc would get me to jump ship from those apps.  That and "Voice Boost".

**Comment by blibbler** [comment]
I am also eager for this feature to exist. I have long gone to great lengths to import audio books into Overcast to have the benefit of this feature. I have recently migrated my listening to Plappa, but greatly miss the silence reduction.
